third_party/ABSEIL_MANUAL.md

```diff
@@ -0,0 +1,30 @@
+# Abseil in gRPC
+
+This document explains how to use Abseil throughout gRPC. Note that this isn't
+supposed to explain general usage of Abseil.
+
+## The version of Abseil
+
+gRPC is inteded to use the LTS versions of Abseil only because it simplfies
+dependency management. Abseil is being distributed via package distribution
+systems such as vcpkg and cocoapods. If gRPC depends on the certain version
```

Comment:
"If gRPC depends on the certain version
that aren't registered, gRPC in that system wouldn't build or work." sounds a bit unclear, can you please rephrase?

Reply:
Yeah, I updated it to make it clearer. PTAL.

Reply:
Btw, this is basically a absl-specific clarification of https://github.com/grpc/grpc/tree/master/third_party#guidelines-on-updating-submodules:
*IMPORTANT: whenever possible, try to only update to a stable release of a library (= not to master / random commit). Depending on unreleased revisions makes gRPC installation harder for users, as it forces them to always build the dependency from source and prevents them from using more convenient installation channels (linux packages, package managers etc.)*

Reply:
Indeed. :)